### PR TITLE
Enable all build flavors for UWP github PR actions

### PR DESCRIPTION
**Pull request by chrisfromwork**
We noticed recently that the release flavor was not building correctly. To try and avoid this in the future, this pr adds tasks to build all UWP flavors for prs.

Note: UWP release flavors may fail until the following pr is checked in: https://github.com/BabylonJS/BabylonReactNative/pull/164

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -63,6 +63,260 @@ jobs:
...
+        run: npx gulp buildUWPx86Debug
+        working-directory: ./Package
+      - name: Gulp NuGet Restore Playground
+        run: npx gulp nugetRestoreUWPPlayground
+        working-directory: ./Package
+      - name: Gulp Build x86 Debug Playground (Windows)
+        run: npx gulp buildUWPPlaygroundx86Debug
+        working-directory: ./Package
+
+  build-windows-x86-release:
```
Comment (ryantrem) [inline .github/workflows/pr.yml:79]: These all look like they are basically clones with configuration variations only, is that right? If so, can we leverage the GitHub workflow "matrix strategy" stuff? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Comment (chrisfromwork) [reply .github/workflows/pr.yml:79]: yes. i will look into this

Comment (chrisfromwork) [reply .github/workflows/pr.yml]: we now use a matrix for setup. Please let me know whether you are also wanting to reduce the number of gulp tasks/if you know of examples for providing arguments to gulp tasks

**Review by ryantrem — APPROVED** [review]

**Comment by chrisfromwork** [comment]
resetting android build

**Pull request merged**